rectools/models/implicit_als.py

```diff
@@ -172,7 +178,20 @@ def fit_als_with_features_separately_inplace(
     verbose : int
          Whether to print output.
     """
+    # If model was fitted we should drop any learnt embeddings except actual latent factors
+    if model.user_factors is not None and model.item_factors is not None:
```

Comment:
`if model.user_factors is None or model.item_factors is None` and switch places

Reply:
can't do it, it's not about switching cases. It's only about this particular case

Reply:
my bad